blog/_posts/2022-01-19-learn-scala.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy, Vincent Derouand, and Valérie Pedroni (Scala Center)
+title: "Learn Scala"
+---
+
+Happy new year 2022! The beginning of the year is a good time to refresh 
+your programming skills. We want to use this opportunity to report on what 
+happened on the MOOCs in 2021, and to share our plans for 2022.
+
+## 2021 Retrospective
+
+In 2021, with the release of Scala 3 we updated most of our MOOCs to Scala 3:
+
+- [Functional Programming Principles in Scala],
+- [Functional Program Design],
+- [Parallel Programming],
+- [Programming Reactive Systems] with Akka.
+
+We also kept the Scala 2 versions of these courses online, but we observed 
+that the majority of the learners took the Scala 3 version (e.g., 97% in the 
+case of Functional Programming Principles in Scala).
+
+The only courses that still use Scala 2 are the courses that depend on Spark,
+which does not yet fully support Scala 3 (see below our plans in this regard).
+
+We also published a new course: [Effective Programming in Scala]. This 
+course teaches programmers everything they need to be ready to work in Scala.
+We are proud to report that this course was one of the [100 most popular 
+courses of the year 2021][rank] (out of the 2900 courses created in 2021 on 
+all the MOOCs platform).
+
+In total, in 2021 more than three thousand people completed a Scala online 
+course! Congratulations to them!
+
+We have recently re-organized the website to have a
+[dedicated page](https://docs.scala-lang.org/online-courses) presenting the 
+Scala courses from the Scala Center.
+
+## Plans for 2022
+
+In 2022, we plan to update the Spark courses to Spark 3.2 and Scala 2.13:
+
+- [Big Data Analysis with Scala and Spark],
+- [Scala Capstone Project].
+
+So far, all our courses are self-paced and the assignments grading is fully 
+automated. This model is perfect for autonomous learners, but we believe 
+there is also a need for a complementary model, where learners would get 
+code reviews from experts, and one-to-one sessions with instructors. This 
+year, we plan to publish some of our courses under that complementary model.
+
+## Acknowledgements
+
+The Scala Center is committed to providing high-quality, free, Scala 
+educational material. This work would not be possible without the support of 
+the learners who pay to get a certificate of completion at the end of the 
+courses. We are grateful to them!
```

Comment:
We are grateful to all of certified learners. This work would not be possible without the support....

Reply:
Thank you! Addressed in 430b3bf0.